Solutions/Marain.UserNotifications.Abstractions/Marain/NotificationTemplates/CommunicationTemplates/EmailTemplate.cs

```diff
@@ -0,0 +1,43 @@
+﻿// <copyright file="EmailTemplate.cs" company="Endjin Limited">
+// Copyright (c) Endjin Limited. All rights reserved.
+// </copyright>
+
+namespace Marain.NotificationTemplates.CommunicationTemplates
+{
+    /// <summary>
+    /// Email Object.
+    /// </summary>
+    public class EmailTemplate : ICommunicationTemplate
+    {
+        /// <summary>
+        /// The content type that will be used when serializing/deserializing.
+        /// </summary>
+        public const string RegisteredContentType = "application/vnd.marain.usernotifications.notificationtemplate.emailtemplate.v1";
+
+        /// <inheritdoc/>
+        public string? NotificationType { get; set; }
+
+        /// <inheritdoc/>
+        public string? ETag { get; set; }
+
+        /// <summary>
+        /// Gets or sets the body of the email.
+        /// </summary>
+        public string? Body { get; set; }
```

Comment:
Should we consider a `TemplateFormat` or something similar? Are we happy that dotliquid will be the only template format we'll ever want?

Reply:
Great question Ian. May be in the future yes.

Currently we would only be supporting dotliquid template format for now.